TeamSnapSDK/SDK/Cache/TSPCache.m

```diff
@@ -22,8 +22,8 @@ +(void)setCacheRootPath:(NSURL *)rootPath {
 
 +(NSURL *)cacheRootPath {
     if (!_rootPath) {
-        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
-        _rootPath = [NSURL fileURLWithPath:directory isDirectory:YES];
+        NSString *directory = [NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES) lastObject];
```

Comment:
I think we could get away with just using the Caches directory. I believe we're tolerant enough of the cache disappearing at any time. 🤔